src/foundry/foundry.js/audioHelper.d.ts

```diff
@@ -254,8 +209,7 @@ declare class AudioHelper {
    * @param callback  - The callback function to call with the decibel level.
```

Comment:
`callback(dbLevel)` is missing here

Reply:
that's intentional as the invocation is incorrect